src/css/dark.css

```diff
@@ -110,6 +109,7 @@ input.number:focus {
 		}
 
 		#menubar .menu .options {
+      cursor: pointer;
```

Comment:
although i'm a bit of a hypocrite with the monster PR haha https://github.com/aframevr/aframe-editor/pull/93 though i can't really break that up